decoder/routes.go

```diff
@@ -94,6 +95,32 @@ func saveRouteRecord(db db.DbDetails, route *Route) error {
 		}
 	}
 
+	if len(route.Description) > 255 {
+		defer func(oldDescr string) {
+			route.Description = oldDescr
+		}(route.Description)
+
+		log.Warnf("truncating description for route id '%s'. Orig description: %s",
+			route.Id,
+			route.Description,
+		)
+
+		// this splits on rune boundaries instead of byte boundaries. only
```

Comment:
r := []rune(text)
   trunc := r[:width]

?

Reply:
yeah, I didn't do that because it causes an allocation, supposedly. But I guess when stop to think about it, who cares, in this spot. I'll change it.